web_fix_modules_load/static/src/js/session.js

```diff
@@ -0,0 +1,127 @@
+odoo.define("web_load_translations_fix.Session", function (require) {
+    "use strict";
+
+    var Session = require("web.Session");
+    var core = require("web.core");
+    var _t = core._t;
+
+    /**
+     * Override session manager to change how modules' are loaded: use ids instead of names.
+     *
+     */
+    Session.include({
+        _modules_info: function () {
+            return odoo._modules_info;
+        },
+        _module_ids: function (names) {
+            const mod_names = names ? names : this.module_list;
+            const info = this._modules_info();
+            const ids = [];
+            _.each(mod_names, function (name) {
+                if (info[name]) {
+                    ids.push(info[name].id);
+                }
+            });
```

Comment:
nitpicking: vanilla js is more powerful nowadays, making tools like this a bit pointless. The trend is to favour vanilla js code, which is usually easier to read

```suggestion
            const ids = mod_names.filter(name => info[name]).map(name => info[name].id);
```